src/PaymentPart/Output/OutputInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Sprain\SwissQrBill\PaymentPart\Output;
+
+use Sprain\SwissQrBill\QrBill;
+
+interface OutputInterface
+{
+    public function getQrBill(): ?QrBill;
+
+    public function getLanguage(): ?string;
+
+    public function getPaymentPart();
```

Comment:
What type it is return value?

Reply:
It varies, depending on the actual implementation – could be a string, could be an instance of some pdf library, etc.
I was torn what to do here. Better suggestions are welcome.